client/components/AllReviews.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchReviews } from '../store/reviews';
```

Comment:
since we're exporting reviews from `/store/index.js`, we can just import from `../store`